domain 0.10.2 built failed after upgrading

Hi, @partim. Thank you for maintaining this library.

I noticed that `0.10.2` is broken after upgrading:

```shell
error[E0433]: failed to resolve: could not find `zonetree` in the crate root
   --> /home/xuanwo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/domain-0.10.2/src/net/xfr/protocol/interpreter.rs:8:12
    |
8   | use crate::zonetree::types::ZoneUpdate;
    |            ^^^^^^^^ could not find `zonetree` in the crate root
    |
note: found an item that was configured out
   --> /home/xuanwo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/domain-0.10.2/src/lib.rs:177:9
    |
177 | pub mod zonetree;
    |         ^^^^^^^^
    = note: the item is gated behind the `unstable-zonetree` feature

error[E0433]: failed to resolve: could not find `zonetree` in the crate root
   --> /home/xuanwo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/domain-0.10.2/src/net/xfr/protocol/iterator.rs:9:12
    |
9   | use crate::zonetree::types::ZoneUpdate;
    |            ^^^^^^^^ could not find `zonetree` in the crate root
    |
note: found an item that was configured out
   --> /home/xuanwo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/domain-0.10.2/src/lib.rs:177:9
    |
177 | pub mod zonetree;
    |         ^^^^^^^^
    = note: the item is gated behind the `unstable-zonetree` feature

```

Would you consider releasing a `0.10.3` to address this? Thanks in advance.

--- a/src/net/xfr/mod.rs
+++ b/src/net/xfr/mod.rs
@@ -2,6 +2,7 @@
     not(feature = "unstable-xfr"),
     doc = " The `unstable-xfr` feature is necessary to enable this module."
 )]
+#![cfg(feature = "unstable-xfr")]
 // #![warn(missing_docs)]
 // #![warn(clippy::missing_docs_in_private_items)]
 //! XFR related functionality.

